app/views/spree/admin/payment_methods/edit.html.haml

```diff
@@ -12,7 +12,7 @@
   %fieldset.no-border-top
     = render partial: 'form', locals: { f: f }
     .one.column &nbsp;
-    = render partial: 'spree/admin/shared/hubs_sidebar', locals: { f: f, klass: :payment_method }
+    = render partial: 'spree/admin/shared/hubs_sidebar', locals: { f: f, klass: :payment_method, mode: :edit }
```

Comment:
do you really need mode? if editing and there's only one hub you can force the check as well, even if it is already selected. the code would be simpler.

Reply:
I have though about that too. My mind was split. I removed the mode variable and reverted the updated templates.

Reply:
oh, so it just works without the mode thing, definitely better!!! #simplicity